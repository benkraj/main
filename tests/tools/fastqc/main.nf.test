nextflow_pipeline {

    name "Test --tools fastqc"
    script "main.nf"

    test("Should run without failures") {

        when {
            params {
                outdir = "$outputDir"
                test = true
                tools = 'fastqc'
            }
        }

        then {
            assert workflow.success
            assert new File("$outputDir/fastqc/test_fastqc.zip").exists()

            assert snapshot(
                "$outputDir/fastqc/test_fastqc.html",
                "$outputDir/fastqc/versions.yml"
            ).match()
        }

    }
}
