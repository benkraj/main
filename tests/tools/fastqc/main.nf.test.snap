{
    "Should run without failures": {
        "content": [
            "test_fastqc.html:md5,0dd01ef6e8655a8524dd3e6357ce9611",
            "versions.yml:md5,b12a5aa87e5bddcf730746e96b769cf0"
        ],
        "timestamp": "2023-01-25T15:38:10+0000"
    }
}